config/default_records/submission_templates/012_scrna_core_library_prep_submission_templates.wip.yml

```diff
@@ -0,0 +1,40 @@
+# Submission templates for the aggregation and library prep parts of the scRNA Core pipeline.
+---
+Limber-Htp - scRNA Core Chromium GEM-X 5p Aggregation:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_scrna_core_chromium_gem_x_5p_aggregation"]
+    product_line_name: Short Read
+    product_catalogue_name: scRNA Core
+Limber-Htp - scRNA Core Library Prep:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_scrna_core_library_prep"]
+    product_line_name: Short Read
+    product_catalogue_name: scRNA Core
+Limber-Htp - scRNA Core Library Prep - Pool:
+  submission_class_name: "LinearSubmission"
+  related_records:
+    request_type_keys: ["limber_scrna_core_library_prep", "limber_multiplexing"]
+    product_line_name: Short Read
+    product_catalogue_name: scRNA Core
+Limber-Htp - scRNA Core Library Prep - NovaSeqX paired end sequencing:
```

Comment:
I don't actually remember having to set up these combination submission templates in the past, was it done in some common code before? Maybe that was pre-record loader...

Reply:
Yes, it's still there in `limber.rake` (run by `rake limber:setup`).

I decided not to use it because a) I think we want to replace it with record loader, and b) the HiSeq options aren't relevant any more (planning to create a story to get rid of HiSeq options).